qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java

```diff
@@ -146,4 +146,38 @@ public void clickOnSaveUserAccountButton() {
 	public void systemAddsUserAccount() {
 		assertTrue(textExists("Account Saved Successfully"));
 	}
+
+	@And("a user enters person details in the user account form") 
+	public void enterPersonalDetails() {
+		userAccountPage.enterPersonalDetails("Clerk", "Data");
+		userAccountPage.selectGender();
+	}
+
+	@And("a user enters user account details in the user account form")
+	public void enterUserAccountDetails() {
+		userAccountPage.clickOnAddUserAccount();
+		userAccountPage.setUsername(CLERK);
```

Comment:
We can use a random user name.

Reply:
ts randomly generated already, take a look at `private static final String CLERK = RandomStringUtils.randomAlphabetic(8);`